randovania/data/json_data/prime1/Chozo Ruins.txt

```diff
@@ -1395,12 +1397,16 @@ Asset id: 4216627403
               Any of the following:
                   Space Jump Boots
                   Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-          Wave Beam and After Hall of the Elders Bomb Slots
+          All of the following:
+              Wave Beam and After Hall of the Elders Bomb Slots
+              Charge Beam or Combat (Intermediate)
           Morph Ball Bomb and Morph Ball and Complex Bomb Jump (Expert)
   > Door to Elder Hall Access
       Trivial
   > Pickup (Energy Tank)
-      Ice Beam and After Hall of the Elders Bomb Slots
+      All of the following:
+          Ice Beam and After Hall of the Elders Bomb Slots
+          Charge Beam or Combat (Intermediate)
```

Comment:
This change is also unnecessary. The change should be applied from Crossway Access South to bomb slots event, since thats when you have to fight the ghosts.

Reply:
Ghosts are required to be fought in order to use the bowling statue. Not the bomb slots. Only the wave and ice bomb slots need bowling statue in order to be used. Plasma bomb slot can be used while ghosts are still alive as it just moves the statue. Hence why I tied the charge requirement to the individual bomb slots instead of the room in general.